README.md

```diff
@@ -63,6 +63,18 @@ If you need to specify a condition for the update, pass it as an Arel query:
 MyRecord.upsert({id: 1, wisdom: 3}, arel_condition: MyRecord.arel_table[:updated_at].lt(1.day.ago))
 ```
 
+The instance `#upsert` can also take keyword arguments to specify a condition, or to limit which attributes to upsert
```

Comment:
The instance _method_

Reply:
In order to speed things up, I'll add that myself after merge.

Reply:
Thanks!